frontend/src/components/Reports/routine/RejectionReport.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from 'react';
+import {
+    Form,
+    FormLabel,
+    Grid,
+    Column,
+    Section,
+    Button,
+    Loading,
+  } from "@carbon/react";
+import { FormattedMessage, useIntl } from 'react-intl';
+import "../../Style.css";
+import { AlertDialog } from "../../common/CustomNotification";
+import CustomDatePicker from "../../common/CustomDatePicker";
+import config from "../../../config.json";
+
+
+const RejectionReport = () => {
+  const intl = useIntl();
+  const [loading, setLoading] = useState(false);
+  const [notificationVisible, setNotificationVisible] = useState(false);
+  const [reportFormValues, setReportFormValues] = useState({
+    startDate: null,
+    endDate: null
+  });
+
+
+
+  function encodeDate(dateString) {
+    if (typeof dateString === "string" && dateString.trim() !== "") {
+      return dateString.split("/").map(encodeURIComponent).join("%2F");
+    } else {
+      return "";
+    }
+  }
+
+  const handleDatePickerChangeDate = (datePicker, date) => {
+    let updatedDate = encodeDate(date);
+    let obj = null;
+    switch (datePicker) {
+      case "startDate":
+        obj = {
+          ...reportFormValues,
+          startDate: updatedDate,
+        };
+        break;
+      case "endDate":
+        obj = {
+          ...reportFormValues,
+          endDate: updatedDate,
+        };
+        break;
+      default:
+    }
+    setReportFormValues(obj);
+  };
+
+  const handleSubmit = () => {
+    setLoading(true);
+
+    const baseParams = 'report=sampleRejectionReport&type=indicator';
+
+    const baseUrl = `${config.serverBaseUrl}/ReportPrint`;
+    const url = `${baseUrl}?${baseParams}&upperDateRange=${reportFormValues.startDate}&lowerDateRange=${reportFormValues.endDate}`;
```

Comment:
```suggestion
    const url = `${baseUrl}?${baseParams}&upperDateRange=${reportFormValues.endDate}&lowerDateRange=${reportFormValues.startDate}`;
```

Hello @manishjha-04 

IG the backend treats `upperDateRange` as `endDate` and `lowerDateRange` as `startDate`

just a minor suggestion

cc: @mozzy11

Reply:
Thanks for catching this @konavivekramakrishna .